pizzabonus/pizzabonus.cpp

```diff
@@ -0,0 +1,74 @@
+#include <eosio/eosio.hpp>
```

Comment:
If you are including `<eosio/eosio.hpp>` in `pizzabonus.h`, it's not necessary to include it again in this file.